Makefile

```diff
@@ -273,4 +273,4 @@ generate-schemas:
 # static files
 
 $(STATIK_FILES): go.mod docs/content/en/schemas/*
-	hack/generate-statik.sh
```

Comment:
typo?

Reply:
ah yep i meant to change this back. this is my workaround so i don't have to collect licenses every time i run `make` locally :)